eth/plasma.go

```diff
@@ -145,19 +145,15 @@ func (plasma *Plasma) GetDeposit(nonce *big.Int) (plasmaTypes.Deposit, *big.Int,
 		return plasmaTypes.Deposit{}, nil, false
 	}
 
-	// check the finality bound
-	ethBlockNum, err := plasma.client.LatestBlockNum()
+	// check the finality bound based off pegged ETH block
+	ethBlockNum, err := plasma.ethBlockPeg(tmBlock)
 	if err != nil {
-		plasma.logger.Error(fmt.Sprintf("failed to retrieve the latest ethereum block number { %s }", err))
```

Comment:
This is just a suggestion, but I think we should consider formatting our nested error messages as specified by [golang](https://github.com/golang/go/wiki/CodeReviewComments#error-strings) and keep the beginning of these errors with lower case.

Reply:
Agreed but this line seems to suggest it doesn't apply in this logging case:

`This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.`

Reply:
I think that would be for logging regular messages and not error messages since in the example they gave, it is an error message being logged but they still require lower case